app/models/host_status/configuration_status.rb

```diff
@@ -22,15 +22,17 @@ def out_of_sync?
     end
 
     def expected_report_interval
-      (reported_origin_interval + default_report_interval).minutes
+      (reported_origin_interval || default_report_interval).minutes
```

Comment:
@nadjaheitmann this needs to be `to_i` before `.minutes`

Reply:
Actually the best would be
`(reported_origin_inteval.presence || default..).to_i`

Reply:
https://github.com/theforeman/foreman/pull/9045 

Can we add a test to catch that?

Reply:
Feel free to :) It should be a simple test for the `configuration_status`